arch/arm/src/nrf52/nrf52832_errdata.c

```diff
@@ -154,4 +160,6 @@ void nrf52832_errdata_init(void)
   nrf52832_errdata_16();
 
   nrf52832_errdata_66_temp();
+
+  nrf52832_errdata_102();
```

Comment:
Can we read the silicon rev information and then conditionally apply these.  This workaround is not needed for rev2 and cuts the sensitivity in half.

Reply:
Is the VARIANT FICR register equivalent to revision? (has values for AAAA, AAAB, etc.). The errata document says something like:
"This anomaly applies to IC Rev. Revision 1, build codes QFAA-Bx0, CIAA-Bx0, QFAB-Bx0."

Reply:
Ah, just found it: https://infocenter.nordicsemi.com/topic/comp_matrix_nrf52832/COMP/nrf52832/ic_revision_overview.html